### ISSUE TITLE
[A11y] Address A11y issues in Applications

### ISSUE THREAD

**Issue by jenny-s51** [opendatahub-io/odh-dashboard#1102]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

- Applications -> Enabled -> Launch Jupyter -> Administration -> Start server progress modal: Scrollable region must have keyboard access

![](https://issues.redhat.com/secure/attachment/12809449/12809449_image-2023-01-20-13-47-24-138.png)

-  Applications -> Enabled -> Launch Jupyter -> Administration -> Start server progress modal: Document should not have more than one banner landmark

![](https://issues.redhat.com/secure/attachment/12809448/12809448_image-2023-01-20-13-48-46-261.png)

- Applications -> Enabled -> Launch Jupyter -> Administration -> Stop all servers modal: Document should not have more than one banner landmark

![](https://issues.redhat.com/secure/attachment/12809447/12809447_image-2023-01-20-13-49-38-954.png)

### Expected Behavior

Resolve axe a11y violations.
